atlas/src/shared/components/Carousel/Carousel.tsx

```diff
@@ -70,7 +69,7 @@ const Carousel: React.FC<CarouselProps> = ({
         arrows={(arrows as unknown) as { prev: string; next: string }}
         {...gliderProps}
       >
-        <Track padding={trackPadding}>{children}</Track>
+        <Track>{children}</Track>
```

Comment:
Since we removed the padding prop here, let's get remove it from all the components that used it

Reply:
It's still used though, we're passing it to the `Container`. I've removed it from the `Track` component because it wasn't really used there, it was passed directly to the div element which didn't have any effect AFAIK

Reply:
I somehow missed that, thanks